modules/side-panel/react/lib/SidePanel.tsx

```diff
@@ -166,10 +166,9 @@ export default class SidePanel extends React.Component<SidePanelProps, SidePanel
         role="region"
         aria-orientation="vertical"
         padding={padding}
-        onBreakpointChange={this.handleResize}
```

Comment:
Why is this appearing as a delete (it is present in the file)

Reply:
`handleResize` is getting enabled in `componentDidMount`. It wasn't used at all in `SidePanelContainer` so this line wasn't doing anything.